sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/batch/RecognizeLinkedEntitiesBatchStringDocuments.java

```diff
@@ -36,7 +36,7 @@ public static void main(String[] args) {
 
         // Recognizing linked entities for each document in a batch of documents
         AtomicInteger counter = new AtomicInteger();
-        for (RecognizeLinkedEntitiesResult entitiesResult : client.recognizeLinkedEntitiesBatch(documents, "en")) {
```

Comment:
It would be good to show these examples with valid TextAnalyticsRequestOptions options now that we don't have the other overload.